src/components/NonProfit/ContactComponent.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+  ChakraProvider,
+  Flex,
+  Button,
+  Modal,
+  ModalBody,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  Icon,
+  Image,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+  useDisclosure,
+} from "@chakra-ui/react";
+import chakraTheme from "@/styles/chakraTheme";
+import { useState, useEffect } from "react";
+export default function ContactComponent() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [firstName, setfirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const [valid, setValid] = useState(false);
+  const colors = { true: "#2352A0", false: "#D9D9D9" };
+  useEffect(() => {
+    if (firstName !== "" && lastName !== "" && email !== "" && message !== "") {
```

Comment:
Can you modify this so that you can only send a message if the email is a valid email (use zod or regex)